docs/examples/sock-shop.md

```diff
@@ -247,27 +282,65 @@ spec:
 
 It describes the application and includes a label derived from the URL, path, and branch to prevent multiple applications from referencing the same source within git.
```

Comment:
could describe why we prevent multiple applications referencing the same source within git. Not essential for merge.

Reply:
That's a good idea but I think it would be good to wait until we've finalized the `remove` behavior (as that's where things go haywire and we can explain the various scenarios at that point).

Reply:
I'm happy with that